### DISCUSSION TITLE
MORE PROXIES FOR PEOPLE WHO NEED IT

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1320] [Exploits JOIN TITANIUM NETWORK TO SHARE PLEASE]
https://gsdfghhihini.quilmes.gob.ar/ Aresenic proxy
 
Buypass | not quite proxy | lasts 3 minutes, doesn't bypass network filters
https://playful-meringue-fb58bc.netlify.app/

If the links above are blocked, just use another proxy to unblock those blocked sites
https://j.xj2.workers.dev/----https://buypass.bypassi.com/

Taco Proxy
https://iwox73-8080.csb.app/
https://dc8371-8080.csb.app/

Breakium | web browser
https://honda8920.honkeykong.org/

FlowOS | web browser
https://mathchamps.net/
https://maths4you.net/

Anura | web browser | Click "normal user"
https://anura.pro
https://anura.mercurywork.shop/ (deprecated)

Glype | proxy | May or may not work from time to time
https://proxy-server.azurewebsites.net/index.php

NEW! miniProxy
NEW https://www.4digitalsignage.com/content/showtimes/beltway/miniProxy.php

Aero Proxy
https://rf425d-8080.csb.app/

Node Unblocker
https://zp5l0r-8080.csb.app/

**Comment by S-PScripts** [comment]
check the links beforehand some don't work

**Reply by ghost** [reply]
They all work for me though

**Comment by ghost** [comment]
all blocked by contentkeeper :(

**Comment by awsome-webdev** [comment]
holy fucking shit anura is GOLD

**Reply by ghost** [reply]
thx but you should be cursing more to the young audiences so they replace their Generation Alpha vocabulary

**Reply by awsome-webdev** [reply]
sry

**Reply by ghost** [reply]
its alright just reminding

**Comment by schoolexploitkid** [comment]
why tf does glype turn stuff into chinese, i can read it but its goofy for some users

**Comment by Luigi-Master** [comment]
For the people who have Securly, none of these work

**Comment by WalkTheEarth** [comment]
WORKING ON LIGHTSPEED:
https://proxy-server.azurewebsites.net/index.php
